app/Http/Controllers/SearchController.php

```diff
@@ -138,13 +138,6 @@ function (Builder $query) use ($escaped_term) {
 				if (AccessControl::is_logged_in()) {
 					$album['owner'] = $album_model->owner->username;
 				}
-				if ($this->readAccessFunctions->album($album_model) === 1) {
```

Comment:
This is going to conflict with `more-refactoring`...

Reply:
yep. which is why I would like to merge the other one before.

Reply:
fixed.